Updated content for Contact the Microsoft Edge DevTools team (#1788)
MicrosoftDocs/edge-developer: microsoft-edge/devtools-guide-chromium/contact.md

ms.topic: conceptual
ms.prod: microsoft-edge
ms.technology: devtools
ms.date: 03/07/2022
---
# Contact the Microsoft Edge DevTools team

Send your feedback to help improve Microsoft Edge DevTools.  Contact the Microsoft Edge DevTools team to submit feature requests or bug reports, or ask questions about Microsoft Edge DevTools.

There are several ways to get in touch with the Microsoft Edge DevTools team:

* To quickly report bugs from DevTools, use the **Send feedback** window.
* To submit feature requests, questions, and have longer conversations with the team, use the **DevTools** repo.

> * To submit cross-browser suggestions, use **The Web Web Want** initiative.

Reviewer: ```suggestion
* To submit cross-browser suggestions, use the **Web Web Want** initiative.
```

> * Using **The Web We Want** initiative to submit cross-browser suggestions.

Reviewer: ```suggestion
* Using the **Web We Want** initiative to submit cross-browser suggestions.
```

Reviewer: And maybe all "ing" such as "using" to "use" and "clicking" to "click" ...

Reviewer: Or, as suggested by Michael, start with the goal such as "To submit cross-browser suggestions, use the **Web We Want** initiative." and "To quickly report bugs from DevTools, use the **Sendfeedback** window."

Author: Sounds great. Will address in a new commit.

* Using the **edge-developer** repo to submit documentation issues.

<!-- ====================================================================== -->
## Use the Send feedback window

To report a bug or a feature request while you are using DevTools, click the **Send feedback** icon or press `Alt`+`Shift`+`I` (Windows, Linux) or `Option`+`Shift`+`I` (macOS).  This approach creates a tracked issue for the Microsoft Edge development team.

![Screenshot of Microsoft Edge, with DevTools open, and the Send feedback icon in DevTools' toolbar](media/devtools-send-feedback.png)

> Clicking the **Send feedback** icon opens a **Send feedback** window which you can use to describe the bug you found or the feature request you have.

Reviewer: To open a window where you describe the bug or feature request, click **Send feedback**.

Author: The paragraph before the image already says to click the send feedback button. So this feels a bit like a repetition. I know how to fix this: I can re-structure this as a bullet list (first click button, then enter info in window), and then remove the first image, and only have one that shows both the button to click and the feedback window to use.

![Screenshot of the Send feedback window, which appeared above Microsoft Edge](media/devtools-send-feedback-window.png)

<!-- ====================================================================== -->
## Use the DevTools repo

To have longer discussions with the DevTools team and community about bugs or features, use the DevTools repository on GitHub.

You can [create a new issue](https://github.com/MicrosoftEdge/DevTools/issues/new/choose) on the repo or [search for existing issues](https://github.com/MicrosoftEdge/DevTools/issues) and join conversations.

> Using the DevTools repo is useful to have longer conversations, especially when you need more than report a bug. On the repo, you can share ideas and tips with others, explain your workflows and use cases in details with the team, and generally have more meaningful interactions with the community.

Reviewer: ```suggestion
Using the DevTools repo is useful to have longer conversations, especially when you need to do more than report a bug. In the repo, you can share ideas and tips with others, explain your workflows and use cases in detail with the team, and generally have more meaningful interactions with the community.
```

Author: Thanks! Addressed in my next commit.

> Using the DevTools repo is useful to have longer conversations, especially when you need to do more than report a bug. In the repo, you can share ideas and tips with others, explain your workflows and use cases in detail with the team, and generally have more meaningful interactions with the community.

Reviewer: ```suggestion
Use the DevTools repo to have longer conversations, especially when you need to do more than report a bug. In the repo, you can share ideas and tips with others, explain your workflows and use cases in detail with the team, and generally have more meaningful interactions with the community.
```
